server/index.js

```diff
@@ -13,8 +14,10 @@ app.use(morgan('dev'));
 app.use(cors());
 app.use(express.json());
 
+app.use('/api', router);
```

Comment:
it is probably better to use the relative path "/reviews" to be more specific.

Reply:
change done!